packages/contracts/test/dollar/ERC20Ubiquity.t.sol

```diff
@@ -28,8 +27,9 @@ contract ERC20UbiquityTest is LocalTestHelper {
 
     event Burning(address indexed _burned, uint256 _amount);
 
-    function setUp() public {
-        dollar_manager_addr = helpers_deployUbiquityDollarManager();
+    function setUp() public override {
+        super.setUp();
```

Comment:
Weird that GitHub renders this super in italics. I presume that its normal unicode symbols if it passed the linters but it stood out to me enough to point attention to.